File: Hackathon.UI/src/app/models/Friendship/FriendshipStatus.ts
export enum FriendshipStatus
{
  /**  Предлоджение дружбы отправлено */
  Pending = 1,

  /**  Предложение дружбы принято */
  Confirmed = 2,

  /**  Предложение дружбы отклонено */
  Rejected = 3
}

export interface IFriendship
{
  //Инициатор предложения дружбы
  proposerId: number;

  //Кому адресовано предложение дружбы
  userId: number;
  status: FriendshipStatus;
}

export enum GetOfferOption
{
  Any = 0,
  Outgoing = 1,
  Incoming = 2
}

File: src/Hackathon.Common/Models/Friend/FriendshipStatus.cs
namespace Hackathon.Common.Models.Friend;

/// <summary>
/// Статус дружбы
/// </summary>
public enum FriendshipStatus: byte
{
    /// <summary>
    /// Предлоджение дружбы отправлено
    /// </summary>
    Pending = 1,

    /// <summary>
    /// Предложение дружбы принято
    /// </summary>
    Confirmed = 2,

    /// <summary>
    /// Предложение дружбы отклонено
    /// </summary>
    Rejected = 3
}
